packages/server/modules/index.ts

```diff
@@ -93,7 +95,13 @@ async function getSpeckleModules() {
   const moduleNames = getEnabledModuleNames()
 
   for (const dir of moduleNames) {
-    loadedModules.push(require(`./${dir}`))
+    const moduleIndex = await import(`./${dir}/index`)
+    const moduleDefinition = 'init' in moduleIndex ? moduleIndex : moduleIndex.default
```

Comment:
Finally supporting `export default {}` in module index files, no longer having to do any of this `export = {}` bullshit